### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by stefanpenner**
* proper test setup (no more delegating to a cli)
* test for configuration (inline vs non-inline source map)
* update links package.json

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,25 +19,15 @@ TranspilerFilter.prototype.extensions = ['js'];
...
-     */
+  return transpiler.transform(string, opts).code;
+};
 
-    return result.code;
-  } catch (err) {
-    throw err;
-  }
+TranspilerFilter.prototype.copyOptions = function() {
+  return JSON.parse(JSON.stringify(this.options));
```
Comment (thejameskyle) [inline index.js]: :mask: I can only hope for the [future](https://github.com/dslomov-chromium/ecmascript-structured-clone).

Comment (thejameskyle) [inline index.js]: You could just add https://www.npmjs.com/package/lodash.clonedeep.

**Comment by stefanpenner** [comment]
@sebmck anything i can do to move this along?

File: index.js (hunk 1 above)
Comment (stefanpenner) [inline index.js]: I think this is fine for now, once the future lands we can do that.

If you are extremely set on pulling in clondeep for this, I will doit.

**Pull request merged**